lib/Doctrine/DBAL/Schema/AbstractSchemaManager.php

```diff
@@ -174,13 +174,18 @@ public function listTableColumns($table, $database = null)
      *
      * Keys of the portable indexes list are all lower-cased.
      *
-     * @param string $table The name of the table.
+     * @param string      $table    The name of the table.
+     * @param string|null $database The name of the database
      *
      * @return Index[]
      */
-    public function listTableIndexes($table)
+    public function listTableIndexes(string $table, ?string $database = null)
```

Comment:
This is a BC-break for extending classes, I think you will have to resort to `func_get_args` if you want to make this BC.

Reply:
True, the second argument is a BC break. :disappointed: